entrypoint.sh

```diff
@@ -20,20 +19,20 @@
 
 
 if [ "$1" = "test" ]; then
-    # Install standard test dependencies; YMMV
-    pip --quiet install \
-        .[test] nose mock PyHamcrest coverage
-    exec nosetests ${NAME}
+   # Install standard test dependencies; YMMV
+   pip --quiet install \
+       .[test] nose "PyHamcrest<1.10.0" coverage
+   exec nosetests
 elif [ "$1" = "lint" ]; then
-    # Install standard linting dependencies; YMMV
-    pip --quiet install \
-        .[lint] flake8 flake8-print flake8-logging-format
-    exec flake8 ${NAME}
+   # Install standard linting dependencies; YMMV
+   pip --quiet install \
```

Comment:
cc/ @piercefreeman this is the line where you'll want to pin isort e.g. add a `isort<5.0.0` pin to this pip install